### DISCUSSION TITLE
Thoughts on lowering default watchOptions.aggregateTimeout?

### DISCUSSION THREAD

**Discussion by markjm** [webpack/webpack#15036] [Ideas]
Continuing to look at watching improvements. Right now, webpack defaults to waiting .2 seconds on every recompile. This data seems to not really "show up" in the outputted completed time, or the `stats` metric on https://webpack.github.io/benchmark/.

We had no issues in our large project lowering it to 50ms, and I see Next sets this value to 5ms (https://github.com/vercel/next.js/pull/22418).

Seems like an easy perf win which is currently hidden away behind an uncommon config. I tried to find history on the reason for 200ms, but could not.

Thoughts? @sokra

**Comment by alexander-akait** [comment]
Interesting, maybe you are right and we decrease default value

**Reply by markjm** [reply]
Ill create a PR to lower it by an order of magnitude - we can discuss on the number more there
